video_xblock/static/html/studio_edit.html

```diff
@@ -141,14 +141,14 @@
               <div class="custom-field-list">
 
                 <div class="custom-field-section-label main-section-label">{% trans "Show default transcripts" %}</div>
-                   <input type="checkbox" class="default-transcripts-switch-input" id="default-transcript-onoff-switch">
+                   <input type="checkbox" class="default-transcripts-switch-input" id="default-transcript-onoff-switch" checked>
```

Comment:
Why does `checked` parameter hardcoded? I think it should consider some variable(s)